### ISSUE TITLE
Build of static information database fails

### ISSUE THREAD

**Issue by pverscha** [unipept/make-database#13]
The GitHub action that automatically builds the static information database on the first day of every month fails since July with the following exception:

```
##[error]Exception in thread "main" java.lang.IllegalArgumentException: No enum constant org.unipept.taxons.Taxon.Rank.STRAIN
	at java.base/java.lang.Enum.valueOf(Enum.java:240)
	at org.unipept.taxons.Taxon$Rank.valueOf(Taxon.java:30)
	at org.unipept.taxons.Taxon$Rank.fromString(Taxon.java:51)
	at org.unipept.taxons.TaxonList.parseDumps(TaxonList.java:50)
	at org.unipept.tools.NamesNodes2TaxonsLineages.main(NamesNodes2TaxonsLineages.java:30)
```

**Comment by ninewise** [comment]
Similar to e641efa88914b243d088f0e889736644de2e39a2, 0af33d537fa1afd0f9a5f3f37d3977e58f888fb1 and 05fe7dd1bcba5d1d3ca9b0aa843cc107c5d94a19?

**Comment by pverscha** [comment]
@ninewise Yes, this probably is the same issue. Strain now also appears as a rank throughout input data and is not a valid rank. I'll add it to the list of unknown ranks.

**Comment by pverscha** [comment]
@bmesuere notified me that "strain" was recently added as a new rank to the NCBI taxonomy and is thus a valid rank. We should update the database (and the unipept web application) accordingly

**Comment by pverscha** [comment]
Fixed in #17
